sub-logs/ryan.json

```diff
@@ -0,0 +1,15 @@
+{
+  "owner": {
+    "username": "lazylamps",
+    "email": "lampslazy@gmail.com"
+  },
+
+  "target": {
+    "CNAME": {
+      "name": "ryan",
+      "value": "lazylamps.github.io/Portfolio/"
```

Comment:
Your CNAME value isn't valid, as it has slashes in them. Example of a CNAME record would be cname.example.com